pkg/state/state_test.go

```diff
@@ -9,40 +9,22 @@ func TestUpdateRmdNodeData(t *testing.T) {
 	tcases := []struct {
 		name           string
 		nodeName       string
-		namespaceName  string
 		nodeData       RmdNodeData
-		expectedStates map[string]string
+		expectedStates []string
 	}{
 		{
-			name:          "test case 1 - namespace updated for particular node",
-			nodeName:      "example-node-1",
-			namespaceName: "example-name-space",
+			name:     "test case 1 - data added to empty RmdNodeData struct",
```

Comment:
Another test case or two here? Case where RmdNodeData struct is not empty, node already exists in list etc.